specification/spec.md

```diff
@@ -0,0 +1,113 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# BIDS Stats Models Specification
+
+Editors:
+: [Alejandro de la Vega][] ([University of Texas at Austin][])
+: [Christopher J Markiewicz][] ([Stanford University][])
+
+---
+
+## Abstract
+
+This document contains specifications for writing BIDS Stats Models, a recipe for fitting
+hierarchical statistical models to neuroimaging datasets.
+This specification was developed in the [Brain Imaging Data Structure (BIDS)][BIDS] community
+as BIDS Extension Proposal (BEP) 2.
+
+## Status of this document
+
+The previously released version of this specification was [1.0.0rc0][].
+
+## Introduction
```

Comment:
I think what you're saying in this section is good but maybe this can be merged with `Abstract`-- since we give a broader intro elsewhere, and have a whole website to go alone with this. 

Just trying to cut boilerplate / filler so we can get straight to the juicy stuff.